radiography/src/main/java/radiography/ScanScopes.kt

```diff
@@ -35,8 +35,25 @@ object ScanScopes {
   }
 
   /**
-   * Scans all Composables in all windows that have the `Modifier.testTag` modifier with the given
-   * [testTag].
+   * Scans only composables that have a [`Modifier.testTag`][androidx.compose.ui.platform.testTag]
```

Comment:
> Scans only composables

This might be confusing, as it sounds like a filter when in fact it includes the entire composable hierarchy for any matched composable. 

I think the confusion comes from the fact that "scan" here means "find roots" but that's not obvious.

Reply:
How about
```suggestion
   * Limits the scope of the scan to start from composables that have a [`Modifier.testTag`][androidx.compose.ui.platform.testTag]
```

Reply:
Done.